src/main/java/com/example/demo/services/ItemService.java

```diff
@@ -112,8 +114,23 @@ public ItemModel delItem(ItemModel item) {
 	}
 
 	@Override
-	public Collection<ItemModel> findItemsValidFilterCategories(String term,List<Integer> categories, int page, int count) throws InvalidDataException{
-		Pageable pgbl=PageRequest.of(page, count);
+	public Collection<ItemModel> findItemsValidFilterCategories(String term,List<Integer> categories, int page, int count, String sort) throws InvalidDataException{
+		Sort sortConf;
+		switch(sort) {
+		case "newness":
```

Comment:
create enum that encapsulates types of item sorting

this switch case can be method in enum